### DISCUSSION TITLE
Where is the addon button in the Trilium web app?

### DISCUSSION THREAD

**Discussion by Vaslo** [TriliumNext/Trilium#4827] [Q&A]
I have loaded the docker version of Trilium as I test out some note taking apps.  I like the markdown preview that Wiki.js has and I want the same in Trilium.  I downloaded a zip addon for Markdown Preview add-on for Trilium and put it in a folder that is mapped to the container.  When I fire up the container, I don't see a button for add-ons anywhere.  I even tried to do this by bashing into the container but I can't seem to run the cli because of the security, even with root.  Someone tell me I'm missing a simple add-on button?  Thanks.

**Comment by zerebos** [comment]
There is no such addon button, you should import the addon into your note tree using the import feature and disabling `Safe Import` as that addon's installation instructions say.

![image](https://github.com/user-attachments/assets/af5b1ddc-3125-4358-a3d7-c976dc5fa82b)

**Reply by Vaslo** [reply]
Sorry, I completely misunderstood how I would see this.  I thought I saw somewhere that there was an addon button but maybe it was hallucinated by a chatbot.  I have used your method and now it works.  I know folks are transitioning to a new version, but for what it's worth I hope this someday just becomes part of the base software.

Thanks so much for your quick answer here!
